src/ui/public/styles/dark-theme.less

```diff
@@ -287,6 +287,14 @@
         }
       }
     }
+    .visualize-spy-container {
+      .ng-binding {
```

Comment:
If the necessary elements aren't styleable without using the ng-binding class, please add another class to make them styleable.